build.sbt

```diff
@@ -153,12 +179,17 @@ lazy val influx =
     .settings(commonSettings:_*)
     .settings(
       name := "precepte-influx",
-      libraryDependencies ++= Seq(
-        // influx deps
-        "com.google.guava" % "guava" % "26.0-jre",
-        "com.squareup.retrofit" % "retrofit" % "1.9.0",
-        "com.squareup.okhttp" % "okhttp" % "2.7.5"
-      ))
+      libraryDependencies += "org.influxdb" % "influxdb-java" % "2.15"
+    )
+    .dependsOn(core)
+
+lazy val applicationinsights =
+  project.in(file("precepte-applicationinsights"))
+    .settings(commonSettings:_*)
+    .settings(
+      name := "precepte-applicationinsights",
+      libraryDependencies += "com.microsoft.azure" % "applicationinsights-core" % "2.4.0-BETA"
```

Comment:
2.4.0 is out of beta btw (cf https://mvnrepository.com/artifact/com.microsoft.azure/applicationinsights-core/2.4.1)

Reply:
:+1: